### DISCUSSION TITLE
Magnify large maps

### DISCUSSION THREAD

**Discussion by euu2021** [freeplane/freeplane#518] [General]
Question originally posted on the docs repository, by @mhechthz:

> I'm using Freeplane for structuring learning content. Therefore maps can become quite large. If I see the entire map I cannot read anything.
>
>Is it possible to apply a magnifying glass to enlage parts of the map without zoomin in and out?

No, there is no magnifying glass. Here are some workarounds:

First, I suggest experimenting with the Map Overview (View->Controls->Map overview).

If it doesn't fit your needs, another idea is using two mindmap views side by side: one for overview and one for closer view.
![image](https://user-images.githubusercontent.com/77707706/182028348-da20d553-390b-4c60-8d82-dfcedade175d.png)

A more advanced solution, which works very well for me, is minimizing nodes. I described this approach [here](https://github.com/freeplane/freeplane/discussions/401#discussioncomment-2845218).

I can think of another solution, but it's kind of messy: format the "selected node" style so it gets bigger:
![image](https://user-images.githubusercontent.com/77707706/182028696-09b733e0-93b4-49f1-a83e-16f37dda2b82.png)

Maybe, with lots of scripting it's possible to turn this last idea in a magnifying glass.

**Comment by euu2021** [comment]
The original question is [this issue](https://github.com/freeplane/docs/issues/63).

**Reply by mhechthz** [reply]
Hmmm .... would be a good solution, but if I go to Map-Overview-> Position -> Lower right (e.g.) nothing is shown. I'm on Windows with the latest version of Freeplane.

**Reply by euu2021** [reply]
You need to activate it under View->Controls->Map overview

**Reply by mhechthz** [reply]
There is no entry to activate the map under view->controls... , but I found it by right click on the background. 

This is for the first approach ok.
